rxjava3/src/main/resources/META-INF/services/io.micronaut.core.convert.TypeConverterRegistrar

```diff
@@ -0,0 +1 @@
+io.micronaut.rxjava3.converters.RxJava3ConverterRegistrar
```

Comment:
if we keep the `@Singleton` annotation in `RxJava3ConverterRegistrar` we don't need this.

Reply:
I have removed it https://github.com/micronaut-projects/micronaut-rxjava3/pull/226/commits/2c002c94c3d595e667300e87cd032839526c8486